pkg/network/server_test.go

```diff
@@ -96,10 +96,12 @@ func TestServerStartAndShutdown(t *testing.T) {
 		require.Eventually(t, func() bool { return 1 == s.PeerCount() }, time.Second, time.Millisecond*10)
 
 		assert.True(t, s.transports[0].(*fakeTransp).started.Load())
+		require.True(t, s.started.Load())
```

Comment:
yep tried to add second call yesterday, but got panics which are not catchable in test. will try again.

Reply:
i meant the case when:
```
s.Start()
s.Shutdown()
s.Start()
```
then it will be :
```
panic: close of closed channel

goroutine 76 [running]:
github.com/nspcc-dev/neo-go/pkg/network.(*Server).relayBlocksLoop(0x14000298800)
	/Users/ekaterinapavlova/Workplace/neo-go/pkg/network/server.go:1619 +0x23c
created by github.com/nspcc-dev/neo-go/pkg/network.(*Server).Start
	/Users/ekaterinapavlova/Workplace/neo-go/pkg/network/server.go:297 +0x30c


Process finished with the exit code 1
```
but we have a comment ```
// Start will start the server and its underlying transport. Calling it twice
// is an error.
```
so we probably don't need such test.